docs/details/logs.md

```diff
@@ -20,25 +23,18 @@ rolls over.
 
 ##### macOS Unified Logging System (ULS)
 
-Currently all of the most recent releases of Santa are built with the macOS
-10.11 SDK. This allows Santa to continue to log to Apple System Logger (ASL)
-instead of ULS. However, on macOS 10.12+ all of the Kernel logs are sent to ULS.
-See the KEXT Logging section below for more details.
+As Santa has been built with macOS 10.12+ SDKs for several releases, Santa's logs
+are also sent to ULS.
```

Comment:
> Santa's logs are also sent to ULS

The event logs (executions, mounts, file events) described above bypass ULS - this line makes it sound like they go to both. Probably worth calling out that the ULS logs are logs _about_ Santa itself instead of the events it's processing.